app/src/main/java/com/infomaniak/drive/data/api/ApiRepository.kt

```diff
@@ -217,6 +218,10 @@ object ApiRepository {
         return callApi(ApiRoutes.unLikeCommentFile(file, commentId), POST)
     }
 
+    fun getShareLink(file: File): ApiResponse<ShareLink> {
+        return callApi(ApiRoutes.shareLink(file), GET)
```

Comment:
We don't do the GET call for ShareLink, do we?

Reply:
Same as above, only used for tests